src/api/parity_set.rs

```diff
@@ -0,0 +1,240 @@
+use api::Namespace;
+use helpers::{self, CallFuture};
+use types::{H256, Address};
+
+
+use Transport;
+
+#[derive(Debug, Clone)]
+/// `Parity_Set` Specific API
+pub struct ParitySet<T> {
+    transport: T
+}
+
+impl<T: Transport> Namespace<T> for ParitySet<T> {
+    fn new(transport: T) -> Self
+        where
+        Self: Sized,
```

Comment:
I don't think the bound is required

Reply:
Correct! Works well too if bound is not present.